packages/frontend/src/commons.ts

```diff
@@ -30,6 +35,48 @@ export const SUPPORTED_CHAIN_ICONS: Record<SupportedChain, Component> = {
     [SupportedChain.Holesky]: markRaw(MuiEthIcon),
 };
 
+export const POPULAR_CHAIN_TOKENS: Record<SupportedChain, Erc20Token[]> = {
```

Comment:
Can we call this `POPULAR_CHAIN_TOKENS`? The tokens are not necessarily "popular" per se, it's just that they are used as a base for many tokens, making them extremely liquid.

Reply:
Addressed in 286c6130e82f12d3cadbc8f73f0c330140d46a21.